### PR TITLE
Update generateUniqueKey JSDoc to match recent changes

### PR DESCRIPTION
**Pull request by pimterry**
The changes in #22 changed generateUUID to generateUniqueKey, made the method synchronous, and updated the README, but didn't completely update the JSDoc.

This PR pulls the correct README documentation into the JSDoc too.

### CODE REVIEW

**Comment by pimterry** [comment]
@jviotti Duh, sorry, yes. Now done.

File: lib/register.coffee [hunk 1]
```diff
@@ -23,16 +23,12 @@ randomstring = require('randomstring')
...
-# @param {Function} callback - callback (error, randomKey)
+# @returns {String} A generated key
 #
 # @example
-# deviceRegister.generateUniqueKey (error, randomKey) ->
-#  	throw error if error?
-# 	# randomKey is a randomly generated key that can be used as either a uuid or an api key
-# 	console.log(randomKey)
+# randomKey = deviceRegister.generateUniqueKey()
+# randomKey is a randomly generated key that can be used as either a uuid or an api key
```
Comment (jviotti) [inline lib/register.coffee]: This was a comment on the original JSDoc example, but its simply a string now. Did you forget to include an extra `#` at the beginning?

**Comment by jviotti** [comment]
@pimterry I left one last minor JSDoc example observation, but I'm approving anyway!

File: lib/register.coffee (hunk 1 above)
Comment (pimterry) [inline lib/register.coffee]: Another good point. The smallest PRs always have the most mistakes :unamused:. Fixed now and merged, thanks.

**Comment by Page-** [comment]
add to whitelist

**Pull request merged**